spring/boot2-actuator-autoconfigure/src/test/resources/application-internalServiceTest.yml

```diff
@@ -0,0 +1,20 @@
+# Use-legacy-processing to use this application-testbed.yml file instead of application.yml
+spring:
+  config:
+    use-legacy-processing: true
+
+armeria:
+  ports:
+    - port: 0
+  internal-services:
+    include: metrics, health
+    port: 0
+    protocols: http
```

Comment:
Do we need protocols for internal-services?
It seems like we serve the internal services for port-based virtual host but we don't accept the protocol when we create a virtual host.

Reply:
> The protocol of the port of virtual host will follow the server’s ports configuration.

Oops, I didn't know about it. Yeah then, it makes sense to have protocols. 😄